indico/modules/events/persons/forms.py

```diff
@@ -40,6 +41,8 @@ def __init__(self, *args, **kwargs):
         from_addresses = ['{} <{}>'.format(session.user.full_name, email)
                           for email in sorted(session.user.all_emails, key=lambda x: x != session.user.email)]
         self.from_address.choices = zip(from_addresses, from_addresses)
+        self.body.description = render_placeholder_info('event-persons-email', event=None, person=None,
+                                                        no_register_link=kwargs.get('no_register_link'))
```

Comment:
pop it from kwargs before calling super to avoid passing the kwarg to the superclass which doesn't even know about it (it won't break anything to pass it but it's cleaner)